### PR TITLE
Fix: keyword arguments to submit

### PR DESCRIPTION
**Pull request by eddiebergman**
Fixes an issue introduced by SMAC 1.2 and was fixed in autosklearn.

* Original issue automl/auto-sklearn#1406
* Fix automl/auto-sklearn#1407

### CODE REVIEW

File: autoPyTorch/utils/single_thread_client.py [hunk 1]
```diff
@@ -61,8 +61,24 @@ def submit(
...
         priority: int = 0,
+        key: Any = None,
+        workers: Any = None,
+        resources: Any = None,
+        retries: Any = None,
+        fifo_timeout: Any = "100 ms",
+        allow_other_workers: Any = False,
+        actor: Any = False,
+        actors: Any = False,
+        pure: Any = None,
```
Comment (ravinkohli) [inline autoPyTorch/utils/single_thread_client.py:72]: I think we also need to add `multi_objectives` to this list as the github actions raise  `E           TypeError: __init__() got an unexpected keyword argument 'multi_objectives'`

Comment (eddiebergman) [reply autoPyTorch/utils/single_thread_client.py:72]: It's unfortunately a seperate problem -_-

Comment (eddiebergman) [reply autoPyTorch/utils/single_thread_client.py:72]: It likely effects `autosklearn` as well so I'll update this PR when I find it

Comment (ravinkohli) [reply autoPyTorch/utils/single_thread_client.py:72]: okay thanks

Comment (eddiebergman) [reply autoPyTorch/utils/single_thread_client.py:72]: Think I fixed it, there are now timeouts occurring though which seem to be another seperate issue.

**Pull request merged**